meinberlin/apps/bplan/serializers.py

```diff
@@ -42,6 +43,7 @@ class BplanSerializer(serializers.ModelSerializer):
         required=False,
         write_only=True,
     )
+    image_alt_text = ImageAltTextField(image_name=_("Tile image"))
```

Comment:
Ahh, now I get why the copyright is not using the ImageCopyrightField.. that is a model field, but we need a serializer field here, so probably best to also add serilalizer.CharField here like copyright field. So sorry I did not get that before!

Reply:
done